### PR TITLE
add sql support

### PR DESCRIPTION
**Pull request by o2sh**
#462

### CODE REVIEW

File: resources/sql.ascii [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{0}   _..------.._
+{0}.-~            ~-.
+{0}|                |
+{0}|"-..________..-"|
+{0}|                |  ____   ___  _
+{0}|                | / ___| / _ \| |
+{0}|"-..________..-"| \___ \| | | | |
+{0}|                |  ___) | |_| | |___
+{0}|                | |____/ \__\_|_____|
```
Comment (spenserblack) [inline resources/sql.ascii:9]:
```suggestion
{0}|                | {1} ____   ___  _
{0}|                | {1}/ ___| / _ \| |
{0}|"-..________..-"| {1}\___ \| | | | |
{0}|                | {1} ___) | |_| | |___
{0}|                | {1}|____/ \__\_|_____|
```

File: src/info/language.rs [hunk 2]
```diff
@@ -236,6 +236,7 @@ define_languages! {
     { Scala, "scala.ascii", define_colors!( [Color::Red, Color::Red] : [Color::TrueColor{ r:223, g:63 ,b:61}, Color::TrueColor{ r:127, g:12 ,b:29} ] ) },
     { Scheme, "scheme.ascii", define_colors!( [Color::White] : [Color::TrueColor{r: 85, g:85, b:85}] ) },
     { Sh, "shell.ascii", define_colors!( [Color::Green] ), "shell" },
+    { Sql, "sql.ascii", define_colors!( [Color::Cyan] ) },
```
Comment (spenserblack) [inline src/info/language.rs:239]:
```suggestion
    { Sql, "sql.ascii", define_colors!( [Color::Cyan, Color::Yellow] ) },
```

**Review by spenserblack — COMMENTED** [review]
What do you think about making the text a different color from the image?

**Comment by o2sh** [comment]
:+1: 

sql2

**Review by spenserblack — APPROVED** [review]
# :tada:

**Pull request merged**